src/llm/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .model import TransformerConfig, Transformer
```

Comment:
nit: Should we call it `wmt_model.py` to make it consistent with the data utility?